src/components/StudyList/StudyList.js

```diff
@@ -65,38 +65,66 @@ const StudyList = ({studyList, loading, activeView, history, myProfile}) => {
   // We track the version off the column state so that we may override it in
   // the future if the schema ever changes
   const existingState = JSON.parse(localStorage.getItem('studyColumns'));
+  const defaultState = {
+    version: 3,
+    columns: [
+      {key: 'kfId', name: 'Kids First ID', visible: true},
+      {key: 'externalId', name: 'phsid/External ID', visible: false},
+      {
+        key: 'anticipatedSamples',
+        name: 'Expected Samples',
+        visible: false,
+      },
+      {
+        key: 'sequencingStatus',
+        name: 'Sequencing Status',
+        visible: false,
+      },
+      {
+        key: 'ingestionStatus',
+        name: 'Ingestion Status',
+        visible: false,
+      },
+      {
+        key: 'phenotypeStatus',
+        name: 'Phenotype Status',
+        visible: false,
+      },
+      {key: 'version', name: 'Version', visible: true},
+    ],
+    sorting: {
+      column: 'name',
+      direction: 'descending',
+    },
+  };
   const [columns, setColumns] = useState(
-    existingState && existingState.version === 2
-      ? existingState
-      : {
-          version: 2,
-          columns: [
-            {key: 'kfId', name: 'Kids First ID', visible: true},
-            {key: 'externalId', name: 'phsid/External ID', visible: false},
-            {
-              key: 'anticipatedSamples',
-              name: 'Expected Samples',
-              visible: false,
-            },
-            {
-              key: 'sequencingStatus',
-              name: 'Sequencing Status',
-              visible: false,
-            },
-            {
-              key: 'ingestionStatus',
-              name: 'Ingestion Status',
-              visible: false,
-            },
-            {
-              key: 'phenotypeStatus',
-              name: 'Phenotype Status',
-              visible: false,
-            },
-            {key: 'version', name: 'Version', visible: true},
-          ],
-        },
+    existingState && existingState.version === 2 ? existingState : defaultState,
```

Comment:
This needs to check that it is at version 3 or it will remove the state everytime.

Reply:
maybe check `existingState.version === defaultState.version` here so we don't need to update it everytime the default state schema changes.